projects/Mallard/src/components/front/index.tsx

```diff
@@ -152,7 +127,103 @@ const Wrapper: FunctionComponent<{
                 {scrubber}
             </View>
             <View style={wrapperStyles.inner}>{children}</View>
-        </View>
+        </>
+    )
+}
+
+const FrontWithResponse = ({
+    frontData,
+    issue,
+}: {
+    issue: Issue['key']
+    frontData: FrontType
+}) => {
+    const color = themeColor.palette.news.bright
+    const [scrollX] = useState(() => new Animated.Value(0))
+    const flatListRef = useRef<AnimatedFlatListRef | undefined>()
+    const { width } = Dimensions.get('window')
+    const cards: FlatCard[] = useMemo(
+        () => flattenCollections(frontData.collections),
+        [], // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Why wouldn't we want `frontData.collections` as a dep here?

Reply:
Short answer: `frontData.collections` is deep so it can't be compared. I assumed that if the collections change that front component will have been thrown away anyway higher up

That said it just hit me `frontData.collections.map({key})=>key)` will work just fine so I'll do that!

Reply:
Cool 👌